Code/GraphMol/DistGeomHelpers/Embedder.h

```diff
@@ -221,6 +221,70 @@ INT_VECT EmbedMultipleConfs(
     bool enforceChirality = true, bool useExpTorsionAnglePrefs = false,
     bool useBasicKnowledge = false, bool verbose = false,
     double basinThresh = 5.0);
+
+struct EmbedParameters {
+  unsigned int maxIterations;
```

Comment:
Java and C# will thank you if you just go ahead and add the getters and setters to the struct.  You'll have to write them anyway.